Source/items.cpp

```diff
@@ -2217,6 +2267,166 @@ _item_indexes RndItemForMonsterLevel(int8_t monsterLevel)
 	});
 }
 
+std::string GetTranslatedItemName(const Item &item)

```

Comment:
Since some names like oils and non magical items are taken directly from the translation table maybe we could return `StringOrView` to avoid string copying in a decent amount of case?